### ISSUE TITLE
pmmeta.sty missing

### ISSUE THREAD

**Issue by vporton** [planetmath/fem2016#2]
The package pmmeta.sty is missing in the repositories. How am supposed to test my pull requests before submitted them if there is no such file?

Where to obtain this file?

**Comment by dginev** [comment]
Right, I'd like to convert the sources to HTML, and also need the pmmeta.sty (+ pmmeta.sty.ltxml, if that is still around)

**Comment by dginev** [comment]
I dug deep with some github archeology and found one version of one binding that was once used for the HTML, namely [planetmath-specials.sty.ltxml](https://github.com/KWARC/LaTeXML/blob/master/lib/LaTeXML/Package/planetmath-specials.sty.ltxml)

Was there also pmmeta.sty.ltxml that lived somewhere else?

**Comment by dginev** [comment]
Actually, another funny find, the pmmeta.sty file seems to have been added to CNL-CIC at some point:
https://github.com/formalabstracts/CNL-CIC/commit/9ed4f163f929a48143bc61362fe769cf24d04e3e

At this point tagging @holtzermann17 to conjure from memory if these two files I linked above are the correct ones for converting the latest PM .tex files.
